### PR TITLE
Add Alan company

### PR DESCRIPTION
**Pull request by olivierpicault**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [x] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: company-profiles/alan.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- Barcelona 🇪🇸
+- Madrid 🇪🇸
+- Brussels 🇧🇪
+- Gent 🇧🇪
+
+
+
+## How to apply
+
+You'll find [here](https://jobs.lever.co/alan) all our current open positions.
```
Comment (KieranRobson) [inline company-profiles/alan.md:61]: A readability suggestion:

```suggestion
You'll find all of our current open positions [here](https://jobs.lever.co/alan).
```

**Comment by dougaitken** [comment]
If you want this for hacktoberfest @olivierpicault it needs to be a full PR by Monday, thanks.

**Review by KieranRobson — APPROVED** [review]
Thanks for the changes @olivierpicault, This is a draft PR, could you clarify whether this is complete or requires more work?

**Comment by olivierpicault** [comment]
> Thanks for the changes @olivierpicault, This is a draft PR, could you clarify whether this is complete or requires more work?

I'm on it. I plan to get done with it by EOD

File: company-profiles/alan.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Alan
+
+## Company blurb
+
+Our mission is to improve health of milions of people and become the number one healthcare partner in Europe
```
Comment (Blightwidow) [inline company-profiles/alan.md:5]:
```suggestion
Our mission is to improve the health of millions of people and become the number one healthcare partner in Europe.
```

File: company-profiles/alan.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Belgium 🇧🇪
+- Sweden 🇸🇪
+- Germany 🇩🇪
+- Luxembourg 🇱🇺
+- Ireland 🇮🇪
+
+
+## Company technologies
+
+Our front-ends are in Javascript, React for web, React-Native for our mobile applications. We also pushing for Typescript. Our back-end is in Python/Flask. We use SQLAlchemy as our ORM. Our database is PostgreSQL. We also use Redis as a messaging system. We are deployed on AWS. All of our code is in two mono-repositories, and we deploy two applications, so no microservices here.
```
Comment (Blightwidow) [inline company-profiles/alan.md:31]:
```suggestion
Our front-ends are in Javascript, React for web, React-Native for our mobile applications. We are also switching completely to Typescript. Our back-end is in Python/Flask. We use SQLAlchemy as our ORM. Our database is PostgreSQL. We also use Redis as a messaging system. We are deployed on AWS. All of our code is in two mono-repositories from which we deploy several applications, so no microservices here.
```

**Review by Blightwidow — COMMENTED** [review]
Great work, just a few edit suggestions

**Comment by olivierpicault** [comment]
> Great work, just a few edit suggestions

🙏

**Review by KieranRobson — APPROVED** [review]

File: company-profiles/alan.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Company size
+
+500+
+
+## Remote status
+
+We are a remote-first company.
+
+You can discover more about our remote-first policy [here](https://medium.com/alan/working-at-alan-from-anywhere-9a6fe381dc0a)
```
Comment (ElPicador) [inline company-profiles/alan.md:15]:
```suggestion
You can discover more about our remote-first policy [here](https://medium.com/alan/working-at-alan-from-anywhere-9a6fe381dc0a).
```

File: company-profiles/alan.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+- Belgium 🇧🇪
+- Sweden 🇸🇪
+- Germany 🇩🇪
+- Luxembourg 🇱🇺
+- Ireland 🇮🇪
+
+
+## Company technologies
+
+Our front-ends are in Javascript, React for web, React-Native for our mobile applications. We are also switching completely to Typescript. Our back-end is in Python/Flask. We use SQLAlchemy as our ORM. Our database is PostgreSQL. We also use Redis as a messaging system. We are deployed on AWS. All of our code is in two mono-repositories from which we deploy several applications, so no microservices here.
```
Comment (ElPicador) [inline company-profiles/alan.md:31]:
```suggestion
Our front-ends are in Typescript, React for web, React-Native for our mobile applications. We switched from Javascript a few months back. Our back-end is in Python/Flask. We use SQLAlchemy as our ORM. Our database is PostgreSQL. We also use Redis as a messaging system. We are deployed on AWS. All of our code is in two mono-repositories from which we deploy several applications, so no microservices here.
```

**Review by ElPicador — COMMENTED** [review]
Thanks @olivierpicault, I've made a few proposals

**Pull request merged**